### ISSUE TITLE
Is there a way to enable dragabliity so you are able to move the boxes around?

### ISSUE THREAD

**Issue by AlbaraaOthman** [iNerdStack/react-native-flexible-grid#29]

**Comment by iNerdStack** [comment]
No, it doesn't have that yet but it will be added in future updates

**Comment by AlbaraaOthman** [comment]
> No, it doesn't have that yet but it will be added in future updates

Thank you for the reply, very appreciated. 

This next bit might be a weird question; do you have any idea when it would be implemented? Of course no rush and no pressure. I'd like to use this package in a project, so just planning for the future :) 

We've tried many draggable packages but none allow for different heights and different widths.

**Comment by iNerdStack** [comment]
Creating draggable items/boxes for uneven heights and widths can be tricky and complex but not impossible, there are just many post arrangement behaviour to consider and will need to be properly tested for different cases.

Right now, its a feature considered for the future and currently the next release is focused mostly on introducing CSS template like feature or arrangement configurations  but if more contributors join the project, the draggable feature can come faster

**Comment by juan-vilas** [comment]
In case it helps I have implemented this feature in here:
https://github.com/juan-vilas/react-native-flexible-grid

I'm planning to create a PR once it's properly tested

**Comment by iNerdStack** [comment]
@juan-vilas awesome, looking forward to it
